ClientTests/SearchEngineManagerTests.swift

```diff
@@ -0,0 +1,32 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+
+import XCTest
+
+class ClientTests: XCTestCase {
+    
```

Comment:
Is this going to be eventually filled in? what's this for?

Reply:
Whoops this was checked in accidentally but I was planning to add unit tests around SearchEngineManager. Should I do that as part of this pull request or a different one (it will require adding unit test bundle to the project)?

Reply:
@joeyg please work on this later: https://github.com/mozilla-mobile/focus-ios/issues/758

We need to code freeze ASAP.

Reply:
Removed that from the PR